OpenRA.Mods.Common/Traits/Player/ProductionQueue.cs

```diff
@@ -549,6 +585,7 @@ public int RemainingTimeActual
 		public bool Done { get; private set; }
 		public bool Started { get; private set; }
 		public int Slowdown { get; private set; }
+		public bool Infinite { get; set; }
```

Comment:
I'm not a fan of allowing outside code to modify this state.  Can we please make this readonly (or at least `private set`)?